sfcasts/setup.md

```diff
@@ -1,3 +1,33 @@
+Hey friends! Welcome back to Episode *2* of our Symfony 7 tutorial. No matter what you do with Symfony, the most important thing you'll use is *services* - little yellow minions that do work in your app. We'll talk about *configuration* for those services as well as *environments*. Environments allow us to influence services so they act differently while developing locally versus on production.
+
+So what exactly *is* a service? That's easy! It's a plain PHP class that does *work*. For example, a `Logger` that helps you log messages is a service. Or a *mailer* that sends emails to your customers. *Or* a database connection object that you use to execute queries to the database. Those are *all* services. Even the controller that handles requests is a service, but it has *super powers*. We'll talk about that later.
+
+This course is titled "Fundamentals" because it's the *foundation*. Everything after this tutorial is just a variation of these themes. *So* to code along with me, download the course code on this page, unzip it, and inside, you'll find a `start/` directory with the same code that you see here. The `README.md` file has everything you need to get this application up and running. I've already completed most of these steps, so I'm going to move on to the last step and run the built-in Symfony web server. To do that, open your terminal and run:
+
+```terminal
```

Comment:
@bocharsky-bw we use a special `terminal-silent` tag for those commands that you don't read in the video

Reply:
yeah, right... ideally, we read all the commands, but during the code blocks adding if you notice that the command was not read - we should add change to `terminal-silent` then 👍